.github/workflows/deploy-community-platform-to-staging.yaml

```diff
@@ -5,6 +5,7 @@ on:
       - main
     paths:
       - "ecosystem/platform/server/**"
+  workflow_dispatch:
```

Comment:
Why did the staging workflow need to change?

Reply:
Just wanted to add the ability to allow manual deployments to staging in exceptional cases.